dmcontent/content_loader.py

```diff
@@ -131,6 +131,7 @@ def create(cls, section):
             return ContentSection(
                 slug=section['slug'],
                 name=section['name'],
+                prefill=section.get('prefill'),
```

Comment:
Should this default to `False`? It will probably get treated like that either way.